plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryClient.java

```diff
@@ -337,4 +362,63 @@ public boolean isAmbiguous()
             return remoteNames.size() > 1;
         }
     }
+
+    static class DestinationTableBuilder
```

Comment:
Can the movement of this class be done in a preparatory first commit? It'll make the diff easier to understand that it's just a move and no code changed.

Reply:
Do you want me to make that change now? Or is it fine to go ahead?

Reply:
It would be better to have that since it makes it easier to make sense of history in the future.

So the move-only changes would get extracted to the first commit.

Reply:
Done.